e2xgrader/server_extensions/formgrader/static/js/e2x_manage_submissions.js

```diff
@@ -0,0 +1,382 @@
+var Submission = Backbone.Model.extend({
```

Comment:
Change filename to manage_submissions.js

Reply:
Changed file name.